
// test('harus melempar kesalahan jika string dilewatkan pada parameter numA',() => {
//     const potensiError = () => {
//         const operandA = '2';
//         const operandB = 2;

//         add(operandA, operandB);
//     }

//     // Asser Throw Error
//     assert.throws(potensiError, Error)
// });